packages/EasyDecision/src/Bridge/Symfony/DependencyInjection/EasyDecisionExtension.php

```diff
@@ -43,5 +44,10 @@ public function load(array $configs, ContainerBuilder $container): void
         $container
             ->autowire(MappingProviderInterface::class, ConfigMappingProvider::class)
             ->setArgument('$decisionsConfig', $config['type_mapping'] ?? []);
+
+        // Register middleware if messenger present
+        if (\class_exists(MessengerPass::class)) {
+            $loader->load('messenger_client.php');
```

Comment:
Can we please name this `messenger_middleware.php`, thank you, otherwise it looks good to me!